src/types/ajv/SyncTrieHashesReq.ts

```diff
@@ -1,7 +1,7 @@
 import { addSchema } from '../../utils/serialization/SchemaHelpers'
 import { AJVSchemaEnum } from '../enum/AJVSchemaEnum'
 
-const SyncTrieHashesRequestSchema = {
+const schemaBroadcastStateReq = {
```

Comment:
@kgmyatthu I guess you accidentally changed SchemaSyncTrieHashesReq to schemaBroadcastStateReq. I have fixed this as part of the latest commit